### PR TITLE
WT-11355 Update test format to always out ANTITHESIS messages to stdout

### PR DESCRIPTION
**Pull request by mickgraham-mongodb**
This is for when tracing is enabled and the messages where going to the ops tracing log instead stdout which is what Antithesis requires.

### CODE REVIEW

File: test/format/wts.c [hunk 1]
```diff
@@ -99,13 +109,14 @@ handle_message(WT_EVENT_HANDLER *handler, WT_SESSION *session, const char *messa
      */
     if ((sap = session->app_private) != NULL && sap->trace != NULL) {
         testutil_check(sap->trace->log_printf(sap->trace, "%s", message));
-        return (0);
-    }
-    if (g.trace_session != NULL) {
+        if (!printf_msg)
+            return (0);
+    } else if (g.trace_session != NULL) {
```
Comment (agorrod) [inline test/format/wts.c:114]: Do these need to be exclusive? i.e: can there be app tracing and global tracing both enabled? I'm not familiar with the functionality..

**Review by agorrod — COMMENTED** [review]
One question, but otherwise lgtm

File: test/format/wts.c (hunk 1 above)
Comment (mickgraham-mongodb) [reply test/format/wts.c:114]: I don't think so. The global is the backup for when the session is not configured for trace. The code previously returned immediately meaning if the first if condition was true, it never got to the second.

**Review by agorrod — APPROVED** [review]
lgtm

**Pull request merged**